src/features/student/views/StudentApcUnavailableView/StudentApcUnavailableView.test.ts

```diff
@@ -0,0 +1,50 @@
+import { useStudentApcAccess } from '@/features/student/composables'
+import { studentHomeRoute } from '@/features/student/routes'
+import StudentApcUnavailableView from '@/features/student/views/StudentApcUnavailableView/StudentApcUnavailableView.vue'
```

Comment:
```suggestion
import StudentApcUnavailableView from './StudentApcUnavailableView.vue'
```
C'est le fichier test associé à la vue et on a décidé que dans l'architecture test et composant seraient toujours dans le même dossier, c'est ok en chemin relatif